### DISCUSSION TITLE
What is your story?

### DISCUSSION THREAD

**Discussion by eddiejaoude** [EddieHubCommunity/support#1749] [Show and tell]
We want to hear your story of how you got into tech and open source 👇 

Things you can include...
- how you got into tech
- how you discovered EddieHub
- how you got into open source
- ...

**Comment by Qodestackr** [comment]
I discovered Eddie when Traversy media invited you. Learnt about open source and now I am a contributor as a technical writer. I feel like a failure in Tech though

**Reply by eddiejaoude** [reply]
Technical writer is great 👍 , this is so important 💥 !

You are not a failure, your GitHub profile looks great 🔥

**Comment by WebShivam** [comment]
I saw you on FreeCodeCamp youtube Channel for the very first time explaining Git and GitHub and I liked your way of explaining there so I directly moved to your YT channel and Subscribed your channel and then after a while I got to know about MLH and EddieHub discord community then l got introduced to Open source and that's how I  got to know about the community 😃 
The community encouraged me to start learning & working on GitHub and I'm really enjoying the process of learning and contributing to Open Source. I loved the way folks here are really active and encouraging to each other 🤩  . There's a lot to learn from all our community members ❤️

**Comment by Qodestackr** [comment]
Thank you. I love your content. I actually have a project as an
organization and private repo. Do you have any video that covers Github
projects and organizations?.

**Comment by Sam-Varghese** [comment]
## How I got into tech

I am a person who didn't like computer science at all till 2019, this was because the school syllabus taught me all the time about excel, making ppts, and the basics of computers, which were not at all of my use back then. When I entered 11th standard, I got python there in my syllabus. But still, due to my very hectic schedule involving physics, chemistry, and math tuitions, all I learnt in 11th standard was just how to print and the basics of functions.

Now when I came in 12th standard, I got time to explore various fields. In order to prove to my CS teacher that I, who never liked CS, could do programming, I started learning python from scratch. As time passed, I liked programming so much that I started programming the whole day. I got interested in programming specifically because my father runs a coaching class and I saw him working hard all day long to satisfy family's needs. I created programs to automate much of his tasks and it looked just magical to make computers do my father's daily works. It was as if I got the world's fastest servant to assist my father.

## How I discovered EddieHub

Gradually I learned python, its libraries like pandas, matplotlib, tkinter etc and qualified Microsoft's exam for python. Felt the need for having a version control system after I got tired of doing `Ctrl+Z` in order to get my previous stable code back, so learnt git, came to Github, got a recommendation by YouTube to look onto Eddie's channel, so joined discord channel ( knew nothing about discord). Putting links of sites other than that of github on Eddie's discord channel made me discover the issues forum situated in eddiehub. I then read the readme and got into EddieHub community.

So till now, I have learned python, HTML, CSS, data structures and algorithms of javascript, SQL,  MongoDB etc. I am presently learning nodejs. My parents and friends were literally shocked after coming to know all that I have learned in CS, because they knew how weak was I in CS just a year ago. I feel happy when I see my class' CS toppers learning basic python, and me having learned and discovered so much.

Thanks for giving me this opportunity to share my journey.

**Comment by Qodestackr** [comment]
This is inspirational. I hope Eddie could share this too. I still feel like
I need to give tech a one genuine try.

**Reply by Sam-Varghese** [reply]
Thank you very much for these words of appreciation. You are really doing great work for the community. You, still feeling like a failure, shows that you are on your way to achieving something very big. Good luck, our blessings are always there with you.

**Comment by Qodestackr** [comment]
It's 3:54 here haven't slept the entire night, and this happens alot to me, from Kenya by the way.

**Comment by ghost** [comment]
I came into tech for those kiddish games when I was in the 1st grade. They got me into mobiles and games. Then my school's web portal taught me computers. Then I started webdev for competition with elder friends who were still using WordPress. That dropped me into wp development. Finally, the lockdown introduced me to discord bot development. Then my dev spirit enriched, I learnt many programming languages. Now since feb here in open source with my project reseter.css
